drp/README.md

```diff
@@ -0,0 +1,7 @@
+# Dummyrepository (drp)
+## Description: dummyrepository.org is a demonstrator designed to show the possibilities for sharing and publishing the capabilities and metadata of (data) repositories. The general goal is to improve the FAIRness of (data) repositories. To show this, the demonstrator needs a few persistent identifiers.
+
+This [w3id](https://w3id.org/dummyrepository) provides a persistent URI namespace for the [dummyrepository demontrator](https://dummyrepository.org) .
```

Comment:
To be clear(er) — @davidlehn is asking whether you might have meant to say *this* in your README...
```suggestion
This [w3id](https://w3id.org/drp) provides a persistent URI namespace for the [dummyrepository demontrator](https://dummyrepository.org) .
```